src/data/reducers/tracks.test.js

```diff
@@ -57,10 +51,7 @@ describe('tracks reducer', () => {
       errorFetching: false,
       finishedFetching: true,
     };
-    expect(tracks(undefined, {
-      type: GOT_TRACKS,
-      tracks: tracksData,
-    })).toEqual(expected);
+    expect(tracks(undefined, actions.fetching.received(tracksData))).toEqual(expected);
```

Comment:
Same comment as in cohorts.test.js about using arbitrary data rather than trying to mimick the data if it isn't being used in the tests.